virttest/qemu_devices/qcontainer.py

```diff
@@ -1748,10 +1748,13 @@ def pcic_by_params(self, name, params):
                   and you are responsible for creating the downstream ports.
         """
         driver = params.get('type', 'ioh3420')
+        pcic_params = {"id": name}
         if driver in ('ioh3420', 'x3130-upstream', 'x3130'):
             bus_type = 'PCIE'
```

Comment:
Thanks your comments and replaced the following code with the above code 

+        if driver == "ioh3420" :
 +            pcic_params["slot"] = params.get("slot")